modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DLFolderDropdownPropsTransformer.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {openConfirmModal, openModal} from 'frontend-js-web';
+
+const _webDAVHTML = (
+	learnMessage,
+	learnURL,
+	portletNamespace,
+	webDavURL
+) => `<div class="portlet-document-library">
+		${Liferay.Language.get('webdav-help')}
+
+		<a href="${learnURL}" target="_blank">${learnMessage}</a>
+		
+		<br/><br/>
+		
+		<div class="form-group input-resource-wrapper">
+			<label class="control-label" for="${portletNamespace}webDavURL">
+				${Liferay.Language.get('web-dav-url')}
+			</label>
+
+			<input class="form-control lfr-input-resource" disabled id="${portletNamespace}webDavURL" name="${portletNamespace}webDavURL" type="text" value="${webDavURL}"/>
+		</div>
+	</div>`;
+
+const ACTIONS = {
+	accessFromDesktop({learnMessage, learnURL, webDavURL}, portletNamespace) {
+		openModal({
+			bodyHTML: _webDAVHTML(
+				learnMessage,
+				learnURL,
+				portletNamespace,
+				webDavURL
+			),
+			onOpen() {
+				const webdavURLInput = document.getElementById(
+					`${portletNamespace}webDavURL`
+				);
+
+				if (webdavURLInput) {
+					webdavURLInput.focus();
+				}
+			},
+			title: Liferay.Language.get('access-from-desktop'),
+		});
+	},
+
+	delete({deleteURL}) {
+		openConfirmModal({
```

Comment:
This also introduces changes. Before we just deleted the folder (without checking if the recycle bin is active or not 🤔 ), now we always show a confirm prompt.... 
```
<liferay-ui:icon
	icon="trash"
	iconCssClass="dropdown-item-indicator-start"
	markupView="lexicon"
	message="delete"
	method="get"
	url="<%= folderActionDisplayContext.getDeleteFolderURL() %>"
/>
```
I'm going to change it as it was before, because thee behaviour from the management toolbar is the same (no confirm prompt appears)